script_creacion_inicial.sql

```diff
@@ -417,4 +417,109 @@ go
 -- TODO - Generar funciones para calcular fact_total en base a ItemxFactura
 -- una vez que este bien armada la tabla VIAJE.
 
-exec GARBAGE.SPMigracion;
\ No newline at end of file
+exec GARBAGE.SPMigracion;
+
+
+
+
+
+
+/******************************************** MIGRACION DE VIAJES  ******************************************/
+
+-----------------Creo tabla viaje-----------------
+
+CREATE TABLE [GARBAGE].[Viaje](
+	[viaje_id] [int] IDENTITY(1,1) NOT NULL,
+	[viaje_auto_id] [int] NOT NULL,
+	[viaje_turno_id] [int] NOT NULL,
+	[viaje_chof_id] [int] NOT NULL,
+	[viaje_cant_km] [numeric](18, 2) NOT NULL,
+	[fecha_hora_ini] datetime NOT NULL,
+	[fecha_hora_fin] datetime NOT NULL,
+	[viaje_cli_id] [int] NOT NULL,
+	[viaje_rendido] [bit] default 0 NOT NULL) --Hay que actualizarlo cuando se hace la tabla de rendiciones
+
+GO
+
```

Comment:
Creo Fede que te falto definir la PRIMARY KEY , por masque "viaje_id" sea identity y not null , definila como primary